submissions/YuraZagor/tiny world pre-oop/index.js

```diff
@@ -49,8 +49,9 @@ cat_woman.saying = cat.saying;
 
 inhabitants.forEach((inhabitant)=>{
   const {species, name, gender, legs, paws, hands, tail, saying, friend} = inhabitant;
-  let textToPrint = `${species}; ${name}; ${gender}; ${legs ? legs : paws}; ${hands ? hands : tail}; ${saying}; ${friend}`;
+  const printArr = [species, name, gender, legs ? legs : paws, hands ? hands : tail, saying, friend]
+
```

Comment:
Why not `.join` in this line? (not a rhetoric question)

Reply:
Also `textToPrint` and `printArr` are quite generic.
What would you call concepts this variables represent in the real world?

E.g. inhabitants are technically objects, but we still give variables that represent them names that are connected with the business domain.

Reply:
not .join()    because we have conditions (ternary)  and they would not be processed if stringified

As per the namings...  **inhabitantRepresentation**  could be less generic I believe

Reply:
inhabitantRepresentation sounds just right for a string data type.
If its content is lest of something then it would be probably inhabitantProperties. Nore the "-s" at the end, this way we denote variable contains multiple things.

Reply:
I see this logic, thanks